en/flight_modes_mc/acro.md

```diff
@@ -27,7 +27,21 @@ Manual control input is required (such as RC control, joystick):
 
```

Comment:
Thanks @MaEtUgR 

I've updated the parameter values in the Acro mode doc here as well. It seems to me that otherwise users will be very disappointed when they try acro mode and it isn't much fun. Because of the existing diagrams in stick input I've suggested that they use the previous default as experienced users. Is this OK?

Further, this really only matters for _manual_ tuning right? I.e. you don't need to worry about default values when autotuning?

Reply:
> I.e. you don't need to worry about default values when autotuning?

Correct

Yes, the reason I lowered the default parameters is that it's much more likely for someone inexperienced to use the defaults and also very likely only for tuning purposes in Acro mode. In that case they rather be the low ones I use for tuning (100°/s) than the high ones we'd usually use on a small FPV racer to be able to turn quickly and flip around. Someone experienced very likely wants to see the plot I explained here:
https://github.com/PX4/PX4-Autopilot/pull/8036#issuecomment-333774139
it essentially allows you to set maximum rate, expo and super expo and see the mathematical result of how input is then interpreted. This way you can dial it in to your liking and flying style:
https://www.desmos.com/calculator/zuqgbayd9g
I still hope we can somehow make that very accessible at least in the docs but ideally also in QGroundcontrol 👀

Reply:
@MaEtUgR Thanks! The calculation and graph are linked below, but there was a rendering issue which I have fixed in https://github.com/PX4/PX4-user_guide/pull/3459/files

So these should be accessible in docs.

For QGC it would probably be nice to include the graph and save the results from that into params automatically. A lot of work though.